contracts/core/Core.sol

```diff
@@ -69,6 +77,15 @@ contract Core is
 
     /* ============ Modifiers ============ */
 
+    // Validate quantity is multiple of natural unit
+    modifier isValidQuantity(uint _quantity, address _setToken) {
```

Comment:
lets mention natural unit in the modifier, `isValidQuantity` of what? we have another "quantity verifier" right below with nonZero check. maybe `isNaturalUnitMultiple`?

Reply:
+1